src/classes/MyHandler.ts

```diff
@@ -1119,6 +1119,6 @@ export = class MyHandler extends Handler {
 
     onTF2QueueCompleted(): void {
         log.debug('Queue finished');
-        this.bot.client.gamesPlayed('tf2-automatic');
+        this.bot.client.gamesPlayed('tf2-automatic', true);
```

Comment:
Why add `true`?

Reply:
The bot will be stuck on playing TF2 after we use "!expand" or this "!delete" command which mostly the cause of the failure to expand or delete because of timeout. Adding true to that so that it forcibly exit TF2 and go back to tf2-automatic.

Reply:
[expand](https://user-images.githubusercontent.com/47635037/84344200-bbcab400-abdc-11ea-8aa9-1955b2e8f8e9.png)